### PR TITLE
Generate code coverage when running tests

### PR DESCRIPTION
**Pull request by mesaugat**
* Code coverage generation after running tests

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+var isparta = require('isparta');
+var istanbul = require('browserify-istanbul');
```
Comment (kabirbaidhya) [inline karma.conf.js]: what a name `istanbul` :smile:. I think now we should create something like `browserify-kathmandu` :wink:

Comment (mesaugat) [inline karma.conf.js]: **istanbul** is a very popular code-coverage reporter. And **isparta** is for ES6. Both are provinces. 

Amazing name for the package though.

Comment (kabirbaidhya) [inline karma.conf.js]: just babaal :+1:

File: package.json [hunk 2]
```diff
@@ -23,7 +27,9 @@
     "vinyl-source-stream": "^1.1.0",
     "watchify": "^3.7.0"
   },
-  "scripts": {},
+  "scripts": {
+    "test": "karma start --single-run --browsers PhantomJS"
```
Comment (kabirbaidhya) [inline package.json]: Let's make it `gulp && karma start --single-run --browsers PhantomJS` so every time we run `npm test` gulp is run before the tests.

Comment (mesaugat) [inline package.json]: We should make another script for running gulp. We won't be making new builds every now and then. I prefer not to add it to `npm test`. We should make `npm build` or something which is a good way to do it.

Comment (kabirbaidhya) [inline package.json]: Agreed.

**Comment by kabirbaidhya** [comment]
:+1:

**Pull request merged**